src/StatPlots.jl

```diff
@@ -34,8 +32,6 @@ include("violin.jl")
 include("hist.jl")
 include("marginalhist.jl")
 include("bar.jl")
-include("shadederror.jl")
```

Comment:
Let me know if this is too heavy-handed, but the syntax has moved out, no?

Reply:
I agree with removing it. It didn't really move anywhere but became superfluous after all the PRs to fix ribbon plots, so now the recommended way is `plot(args...; ribbon = ...)` which is much more standard.